nx-dev/data-access-documents/src/lib/blog.model.ts

```diff
@@ -9,6 +9,7 @@ export type BlogPostDataEntry = {
   reposts: string[];
   updated?: string;
   pinned?: boolean;
+  episode?: number;
```

Comment:
I don't think you need a dedicated frontmatter entry here. A podcast entry should be almost just like a normal blog post and the sorting can happen via the publish date